static/app/views/settings/dynamicSampling/projectsPreviewTable.tsx

```diff
@@ -204,31 +314,68 @@ const SortableHeader = styled('button')`
   gap: ${space(0.5)};
 `;
 
-const ToggleWrapper = styled('div')`
+const Cell = styled('div')`
   display: flex;
-  align-items: center;
-  gap: ${space(0.75)};
-  padding: 0 0 0 ${space(1)};
+  flex-direction: column;
+  gap: ${space(0.25)};
 `;
 
-const PeriodToggle = styled('button')`
-  border: none;
-  background: none;
-  color: ${p => (p['data-is-active'] ? p.theme.textColor : p.theme.disabled)};
-  cursor: pointer;
-  padding: 0;
-  text-transform: uppercase;
-`;
-
-const Cell = styled('div')`
+const FirstCellLine = styled('div')`
   display: flex;
   align-items: center;
-
+  height: 32px;
   &[data-align='right'] {
     justify-content: flex-end;
   }
+  &[data-has-chevron='false'] {
+    padding-left: ${space(2)};
+  }
+`;
+
+const SubProjects = styled('div')`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+  margin-left: ${space(2)};
+  & > div {
+    line-height: 2;
+    margin-right: -${space(2)};
+    padding-right: ${space(2)};
+    margin-left: -${space(1)};
+    padding-left: ${space(1)};
+    border-top-left-radius: ${p => p.theme.borderRadius};
+    border-bottom-left-radius: ${p => p.theme.borderRadius};
+    &:nth-child(odd) {
+      background: ${p => p.theme.backgroundSecondary};
+    }
+  }
+`;
+
+const SubSpans = styled('div')`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+  text-align: right;
+  & > div {
+    line-height: 2;
+    margin-left: -${space(2)};
+    padding-left: ${space(2)};
+    margin-right: -${space(1)};
+    padding-right: ${space(1)};
```

Comment:
```suggestion
 margin: 0 -${space(2)}; 
 padding: 0 ${space(2)};
```

Reply:
One is `space(2)` the other one `space(1)`